### PR TITLE
add endpoint to create multiple project areas at once. 

### PR DESCRIPTION
**Pull request by elsiel23**
all polygons/multipolygons passed from the FE are converted to multipolygons before getting stored in the DB.

### CODE REVIEW

File: src/planscape/plan/urls.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
                         get_scores, list_plans_by_owner,
                         list_projects_for_plan, list_scenarios_for_plan,
                         update_project, favorite_scenario,
-                        unfavorite_scenario)
+                        unfavorite_scenario,create_project_areas_for_project)
```
Comment (strangeties) [inline src/planscape/plan/urls.py:8]: nit: VSCode can automatically sort the imports.

File: src/planscape/plan/views.py [hunk 2]
```diff
@@ -408,6 +408,52 @@ def delete_projects(request: HttpRequest) -> HttpResponse:
...
+        # placeholder project created for 1 forsys with patchmax run)
+        project_id = body.get('project_id', None)
+        if project_id is None or not (isinstance(project_id, int)):
+            raise ValueError("Must specify project_id as an integer")
+
+        # Get the Project, and if the user is logged in, make sure either
+        # 1. the Project owner and the owner are both None, or
+        # 2. the Project owner and the owner are both not None, and are equal.
+        project = Project.objects.get(pk=int(project_id))
+        if not ((owner is None and project.owner is None) or
```
Comment (strangeties) [inline src/planscape/plan/views.py:429]: nit: it looks like this if statement is also used in create_project_areas. Could it be generalized in a separate function?

File: src/planscape/plan/tests.py [hunk 3]
```diff
@@ -804,6 +804,138 @@ def test_good_multipolygon(self):
         self.assertEqual(ProjectArea.objects.count(), 1)
 
 
+class BulkCreateProjectAreaTest(TransactionTestCase):
```
Comment (strangeties) [inline src/planscape/plan/tests.py:807]: nit: are there any tests for checking behavior when user=None, project.owner = None?

**Comment by strangeties** [comment]
Looks fine!
I've added a few nits, but will lgtm since I don't want them to be blocking.

**Review by strangeties — APPROVED** [review]

File: src/planscape/plan/views.py (hunk 2 above)
Comment (elsiel23) [reply src/planscape/plan/views.py:429]: eventually i'd like to deprecate one of these two calls (you can create a single project area by just populating the list with 1 item). let me add a TODO for that instead.

File: src/planscape/plan/tests.py (hunk 3 above)
Comment (elsiel23) [reply src/planscape/plan/tests.py:807]: Done

**Pull request closed without merging**